sound/soc/sof/ipc.c

```diff
@@ -449,14 +456,6 @@ void snd_sof_ipc_msgs_rx(struct snd_sof_dev *sdev)
 }
 EXPORT_SYMBOL(snd_sof_ipc_msgs_rx);
 
-/* schedule work to transmit any IPC in queue */
-void snd_sof_ipc_msgs_tx(struct snd_sof_dev *sdev)
-{
-	if (!sdev->disable_ipc_queue)
-		schedule_work(&sdev->ipc->tx_kwork);
-}
-EXPORT_SYMBOL(snd_sof_ipc_msgs_tx);
-
 /*
```

Comment:
It looks like this function was moved. Why? I was expecting to only add the static keyword and remove the EXPORT_SYMBOL? If the function wasn't moved but git diff shows it like that apologies in advance :)

Reply:
it was moved because it's become static and so it had to be before its use in the file. But actually this function is completely removed two patches later, so, I'll squash them together.